src/checklib.py

```diff
@@ -69,7 +69,15 @@ def verify(self):
     def get_checks(self):
         return self.check_bool(), self.check_detail()
 
-    def add_md(self, md_file_path, img_folder, relative_path_to_img_in_md, item_dict):
+    def add_md(
+        self,
+        md_file_path,
+        img_folder,
+        relative_path_to_img_in_md,
+        item_dict,
+        plot_option=None,
+        fig_size=(6.4, 4.8),
+    ):
```

Comment:
The modifications in this file were made to allow the markdown document to be created using specific arguments from the API.